hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMetrics.java

```diff
@@ -400,6 +433,99 @@ public void incNumListMultipartUploadParts() {
     numListMultipartUploadParts.incr();
   }
 
+  public void incNumTenants() {
+    numTenants.incr();
+  }
+
+  public void decNumTenants() {
+    numTenants.incr(-1);
+  }
+
+  public void incNumTenantCreates() {
+    numTenantOps.incr();
+    numTenantCreates.incr();
+  }
```

Comment:
We could. But the reason I have not added it is because we already increment the total number of tenants on successful tenant creation. Similar to what volume create does:

https://github.com/apache/ozone/blob/fcd016021452d14371e3335982ee0f7cb72fac5c/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeCreateRequest.java#L193-L201

Maybe at some point when we do some metrics improvement where we decided add `incNumVolumeCreateSuccesses` we can do that in the same PR.

Reply:
Update: I'm getting rid of all the success metrics.